db/seedfiles/notice_triggers_seed.rb

```diff
@@ -470,7 +470,71 @@
   },
 ]
```

Comment:
@saip7795 I made a PR which will address this issue. it is ok in this case. When I receive conflict I will make sure to take HEAD changes.